integtest/spec/spec_helper.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+require 'tmpdir'
+require 'fileutils'
+require 'open3'
+
+ENV['GIT_AUTHOR_NAME'] = 'Test'
+ENV['GIT_AUTHOR_EMAIL'] = 'test@example.com'
+ENV['GIT_COMMITTER_NAME'] = 'Test'
+ENV['GIT_COMMITTER_EMAIL'] = 'test@example.com'
+
+RSpec.configure do |config|
+  # Enable flags like --only-failures and --next-failure
+  config.example_status_persistence_file_path = '.rspec_status'
+
+  # Disable RSpec exposing methods globally on `Module` and `main`
+  config.disable_monkey_patching!
+
+  config.expect_with :rspec do |c|
+    c.syntax = :expect
+  end
+end
+
+RSpec.shared_context 'tmp dirs' do
```

Comment:
Are you going to use this context in other files as well as you write more rspec tests?

Reply:
I think I'll use it in all of the context that do conversions, yes. I will need to document it, and probably rename it.